modules/slugs/includes/qtranslate-slug-admin.php

```diff
@@ -40,6 +40,10 @@ function qts_taxonomies_hooks() {
             add_filter( 'manage_' . $taxonomy->name . '_custom_column', 'qts_taxonomy_custom_column', 0, 3 );
         }
     }
+
+    if ( is_plugin_active( 'woocommerce/woocommerce.php' ) ) {
+        add_action( 'woocommerce_after_add_attribute_fields', 'qts_show_add_term_fields' );
+    }
```

Comment:
This is the first time we have such combined integration. Shouldn't we check the WC module state instead?

Reply:
That is exactly why I used the previous hook mechanism...

Reply:
We may check the module status, but eventually the check will be the same.
At least in this way you are covered in the remote case you want to manually disable woocommerce module and still use the slugs.

Reply:
Yes, the module state should be enough. It can be done without hooks, I want to avoid them as much as possible because it makes the maintenance more complicated.

Reply:
done 6513fb44b3c3e45e5bcab871f12e719ca758c486